packages/launchpad/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@packages/launchpad",
+  "version": "0.0.0-development",
+  "private": true,
+  "scripts": {
+    "check-ts": "vue-tsc --noEmit",
+    "build-prod-ui": "cross-env NODE_ENV=production vite build",
+    "clean": "rimraf dist && rimraf ./node_modules/.vite && rimraf dist-e2e && echo 'cleaned'",
+    "clean-deps": "rimraf node_modules",
+    "test": "yarn cypress:run:ct && yarn types",
+    "windi": "yarn windicss-analysis",
+    "cypress:open": "cross-env TZ=America/New_York node ../../scripts/cypress open --project ${PWD}",
+    "cypress:open:ct": "cross-env TZ=America/New_York node ../../scripts/cypress open --component --project ${PWD}",
+    "cypress:run:ct": "cross-env TZ=America/New_York node ../../scripts/cypress run --component --project ${PWD}",
+    "cypress:run:e2e": "cross-env TZ=America/New_York node ../../scripts/cypress run --e2e --project ${PWD}",
+    "dev": "yarn gulp dev --project ${PWD}",
+    "start": "echo 'run yarn dev from the root' && exit 1",
+    "watch": "echo 'run yarn dev from the root' && exit 1"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@cypress/vue": "0.0.0-development",
+    "@graphql-typed-document-node/core": "^3.1.0",
+    "@headlessui/vue": "1.4.0",
+    "@iconify/json": "1.1.368",
+    "@iconify/vue": "3.0.0-beta.1",
+    "@intlify/vite-plugin-vue-i18n": "2.4.0",
+    "@packages/frontend-shared": "0.0.0-development",
+    "@percy/cypress": "^3.1.0",
+    "@testing-library/cypress": "8.0.0",
+    "@urql/core": "2.3.1",
+    "@urql/devtools": "2.0.3",
+    "@urql/vue": "0.4.3",
+    "@vitejs/plugin-vue": "1.2.4",
+    "@vitejs/plugin-vue-jsx": "1.1.6",
+    "@vue/compiler-core": "3.2.6",
+    "@vue/compiler-dom": "3.2.6",
+    "@vue/compiler-sfc": "3.2.6",
+    "@vueuse/core": "5.2.0",
+    "bluebird": "3.5.3",
+    "classnames": "2.3.1",
+    "concurrently": "^6.2.0",
+    "cross-env": "6.0.3",
+    "cypress-file-upload": "^5.0.8",
+    "graphql": "^15.5.1",
+    "graphql-tag": "^2.12.5",
+    "gravatar": "1.8.0",
+    "javascript-time-ago": "2.3.8",
+    "rimraf": "3.0.2",
+    "rollup-plugin-polyfill-node": "^0.7.0",
+    "type-fest": "^2.3.4",
+    "vite": "2.4.4",
+    "vite-plugin-components": "0.11.3",
+    "vite-plugin-icons": "0.6.3",
+    "vite-plugin-optimize-persist": "0.0.5",
+    "vite-plugin-package-config": "0.0.3",
+    "vite-plugin-purge-icons": "0.7.0",
+    "vite-plugin-windicss": "1.2.4",
+    "vite-svg-loader": "2.1.1",
+    "vue": "3.2.6",
+    "vue-i18n": "9.2.0-beta.7",
+    "vue-tsc": "^0.3.0",
+    "vue3-file-selector": "^1.0.1",
+    "windicss": "3.1.4",
+    "windicss-analysis": "^0.3.4",
+    "wonka": "^4.0.15"
+  },
+  "files": [
```

Comment:
This should only need to include `dist`. Otherwise, source files will be unnecessarily included in the binary.